spec/fixtures/files/offline_vm.yml

```diff
@@ -0,0 +1,29 @@
+:apiVersion: kubevirt.io/v1alpha1
```

Comment:
Is this really a yaml file? there are always ":" prefixes

Reply:
It is yml file with `symbols` instead of strings. It is correct

Reply:
thanks